libfwupdplugin/fu-common.h

```diff
@@ -67,7 +67,11 @@ typedef guint FuEndianType;
  * @FU_PATH_KIND_SYSFSDIR_SECURITY:	The sysfs security location (IE /sys/kernel/security)
  * @FU_PATH_KIND_ACPI_TABLES:		The location of the ACPI tables
  * @FU_PATH_KIND_LOCKDIR:		The lock directory (IE /run/lock)
+<<<<<<< HEAD
  * @FU_PATH_KIND_SYSFSDIR_FW_ATTRIB	The firmware attributes directory (IE /sys/class/firmware-attributes)
+=======
+ * @FU_PATH_KIND_LOCKFILE;              The lock file for CrOS systems (IE /run/lock/power_override/fwupd.lock)
+>>>>>>> 608f1be3 (CHROMIUM: inhibit system suspend during update)
```

Comment:
this looks like a bad rebase or merge

Reply:
I had @binaryNewts force push this back to the original commit.